core/storage/exploration/gae_models.py

```diff
@@ -96,6 +96,7 @@ class ExplorationModel(base_models.VersionedModel):
     default_skin = ndb.StringProperty(default='conversation_v1')
     # DEPRECATED in v2.5.4. Do not use.
     skin_customizations = ndb.JsonProperty(indexed=False)
+    image_counter = ndb.IntegerProperty(default=0, indexed=False)
```

Comment:
Move this above line 89. (as below the line 89 all the values are deprecated.)

Reply:
done